noxfile.py

```diff
@@ -24,7 +24,7 @@ def typecheck(session: nox.Session) -> None:
 def tests(session: nox.Session) -> None:
     """Run the unit tests."""
     session.install("--editable", ".[test]")
-    session.run("pytest", "tests/unit", *session.posargs)
+    session.run("pytest", "tests/unit", "-rxXs", *session.posargs)
```

Comment:
```suggestion
    session.run(
        "pytest",
        "tests/unit",
        "-rxXs",  # Show extra summary info for (x)fail, (X)pass, and (s)kipped statuses
        *session.posargs,
    )
```

It's easy to accidentally interpret this as multiple short-form args glommed together instead of one short-form arg and its options.